hazelcast/src/main/java/com/hazelcast/jet/impl/connector/HazelcastReaders.java

```diff
@@ -174,11 +172,8 @@ public int getClassId() {
     }
 
     @Nonnull
-    public static ProcessorMetaSupplier readLocalMapSupplier(@Nonnull String mapName, @Nullable int[] partitions) {
-        return new LocalProcessorMetaSupplier<>(
-                new LocalMapReaderFunction(mapName),
-                partitions
-        ) {
+    public static ProcessorMetaSupplier readLocalMapSupplier(@Nonnull String mapName) {
```

Comment:
so the variant with partition ids will not be available in Jet (CoreAPI), only in SQL?

Reply:
previous PMS was nice and fine-grained - you could specify partition ids per vertex which is exactly what Jet CoreAPI user would like to use.

Reply:
Ah, I see. It was not available before my previous changes. This change just reverts my changes to the functionality available in 5.3.